includes/class-frontend-form-post.php

```diff
@@ -1017,4 +1021,60 @@ public function send_mail_for_guest( $charging_enabled, $post_id, $form_id, $is_
 
         return $response;
     }
+    /**
```

Comment:
Use following for doc block. Always include 1. Summery, 2. `@since` - if new function, 3. `@param`, 4. `@return`. Remove the `$context` param, since we're not using it and update the last param in the hook in __construct, `add_filter( 'get_edit_post_link', [ $this, 'edit_post_link' ], 10, 2 );`
```
/**
 * Enable post edit link for post authors
 * 
 * @since WPUF_SINCE
 * 
 * @param string $url
 * @param int    $post_id
 * 
 * @return string
*/
```